pkgs/development/python-modules/nimbo/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, python3Packages, fetchFromGitHub, awscli }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "nimbo";
+  version = "0.2.2";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "nimbo-sh";
+    repo = "nimbo";
+    sha256 = "0qagib2607qvp5fl71325qpyrpvfhgmk1zix0nbws91n6i3fzc6i";
+    rev = "bfe1e691469011f0fd9c64856a47b07493556b4b"; # up-to-date 2021-04-28
+  };
+
+  propagatedBuildInputs = with python3Packages; [ setuptools boto3 awscli requests click pyyaml pydantic ];
+
+  # nimbo tests require an AWS instance
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Run machine learning jobs on AWS with a single command";
+    homepage = "https://test.pypi.org/project/nimbo";
```

Use the project's homepage. This saves often a click ;-)

```suggestion
    homepage = "https://github.com/nimbo-sh/nimbo";
```